### PR TITLE
Update to new ESH feature

### PR DESCRIPTION
**Pull request by dvanherbergen**
In preparation of https://github.com/openhab/openhab-distro/issues/11

Signed-off-by: Davy Vanherbergen <davy.vanherbergen@gmail.com>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -289,6 +290,11 @@
             <url>https://jcenter.bintray.com/</url>
         </repository>
         <repository>
+            <id>shk-bintray</id>
+            <name>JCenter Repository</name>
```
Comment (maggu2810) [inline pom.xml]: Don't think the name is correct.

Comment (dvanherbergen) [inline pom.xml]: Oops. I think I copy pasted this in a few places...

**Comment by dvanherbergen** [comment]
name updated.

**Comment by kaikreuzer** [comment]
@dvanherbergen Did you also create a similar PR for openHAB 1? (https://github.com/openhab/openhab/tree/master/features)

**Comment by dvanherbergen** [comment]
Yes, https://github.com/openhab/openhab/pull/3703

**Comment by kaikreuzer** [comment]
Ah, cool, thanks!

**Comment by teichsta** [comment]
ftr: https://github.com/openhab/openhab/pull/3703 has been merged. Shall it be added to the 1.8 branch as well?

**Comment by kaikreuzer** [comment]
@teichsta Yes, definitely! Thanks :-)

**Comment by teichsta** [comment]
also cherry picked onto 1.8 …

**Pull request merged**